core/tests/test_utils.py

```diff
@@ -466,6 +466,26 @@ def get_updated_param_dict(
                 obj_type, new_param_dict)
         return new_param_dict
 
+    def get_cache_slug_filepath(self):
```

Comment:
`get_static_asset_url` uses forward slashes since those are paths that are used by the browser, whereas `get_cache_slug_filepath` returns the path to file on a disk, using forward / backward slashes depending on the OS.